package.json

```diff
@@ -62,7 +62,7 @@
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
-    "@openedx/paragon": ">= 21.11.3 < 23.0.0",
+    "@openedx/paragon": ">= 22.0.0 < 24.0.0",
```

Comment:
[consider] _Technically_ breaking in that it drops support for v21; you could probably keep the >= 21.11.3 support to fully defer on risk of breaking change as part of this PR.